.travis.yml

```diff
@@ -17,13 +17,14 @@
 
 language: python
 python:
+  - 3.7
   - 3.6
   - 3.5
```

Comment:
Do we need to test 3.5?  It's support ends in a month or so.

Reply:
i think might remove travis all together since we are using GHA for the tests as well